openpower/configs/linux/skiroot_defconfig

```diff
@@ -322,4 +322,5 @@ CONFIG_DEBUG_CREDENTIALS=y
 # CONFIG_FTRACE is not set
 # CONFIG_RUNTIME_TESTING_MENU is not set
 CONFIG_BUG_ON_DATA_CORRUPTION=y
+CONFIG_PPC_DISABLE_WERROR=y
```

Comment:
This was probably also added by linux-update-config, should be added by buildroot automatically